styles.css

```diff
@@ -0,0 +1,4 @@
+body {
+    background-color: antiquewhite;
```

Comment:
Probeer de volgende keer hsl kleuren te gebruiken.

Reply:
Neen, ik ga niet met HSL werken. Dit is een single comment.